Modules/_sqlite/connection.c

```diff
@@ -2091,7 +2091,14 @@ serialize_impl(pysqlite_Connection *self, const char *name)
                      name);
         return NULL;
     }
-    PyObject *res = PyBytes_FromStringAndSize(data, size);
+#if PY_SSIZE_T_MAX < 9223372036854775807
```

Comment:
To not make readers decyphering what this number is:

```suggestion
#if defined(LLONG_MAX) && PY_SSIZE_T_MAX < LLONG_MAX
```